client/my-sites/plugins/jetpack-plugins-setup/index.jsx

```diff
@@ -540,28 +538,32 @@ class PlansSetup extends Component {
 	}
 }
 
-export default connect(
-	( state, ownProps ) => {
-		const siteId = getSelectedSiteId( state );
-		const selectedSite = getSelectedSite( state );
-		const forSpecificPlugin = ownProps.forSpecificPlugin || false;
-
-		return {
-			sitePlugin: forSpecificPlugin && getPluginOnSite( state, siteId, forSpecificPlugin ),
-			wporgPlugins: getAllWporgPlugins( state ),
-			isRequesting: isRequesting( state, siteId ),
-			requestingInstalledPlugins: isRequestingInstalledPlugins( state, siteId ),
-			hasRequested: hasRequested( state, siteId ),
-			isInstalling: isInstalling( state, siteId, forSpecificPlugin ),
-			isFinished: isFinished( state, siteId, forSpecificPlugin ),
-			plugins: getPluginsForSite( state, siteId, forSpecificPlugin ),
-			activePlugin: getActivePlugin( state, siteId, forSpecificPlugin ),
-			nextPlugin: getNextPlugin( state, siteId, forSpecificPlugin ),
-			selectedSite: selectedSite,
-			isRequestingSites: isRequestingSites( state ),
-			sitesInitialized: hasInitializedSites( state ),
-			siteId,
-		};
-	},
-	{ requestSites, fetchPluginData, installPlugin }
-)( localize( PlansSetup ) );
+export default flow(
+	localize,
+	withInstalledPlugins,
+	connect(
+		( state, ownProps ) => {
+			const siteId = getSelectedSiteId( state );
+			const selectedSite = getSelectedSite( state );
+			const forSpecificPlugin = ownProps.forSpecificPlugin || false;
+
+			return {
+				sitePlugin: forSpecificPlugin && getPluginOnSite( state, siteId, forSpecificPlugin ),
+				wporgPlugins: getAllWporgPlugins( state ),
+				isRequesting: isRequesting( state, siteId ),
+				hasRequested: hasRequested( state, siteId ),
+				isInstalling: isInstalling( state, siteId, forSpecificPlugin ),
+				isFinished: isFinished( state, siteId, forSpecificPlugin ),
+				plugins: getPluginsForSite( state, siteId, forSpecificPlugin ),
```

Comment:
Do we still need this? The new `withInstalledPlugins` now provides a `installedPlugins` prop which should be a replacement for `getPluginsForSite`, no?

Note that this applies to other areas in the code where we start using the `withInstalledPlugins` HoC but we still use the good old Redux selectors for returning plugin data.